pandas/tests/reshape/concat/test_concat.py

```diff
@@ -597,6 +597,15 @@ def test_concat_preserves_extension_int64_dtype():
     tm.assert_frame_equal(result, expected)
 
 
+def test_concat_bool_boolean():
```

Comment:
can you parameterize and also include a bool series here as a test (e.g. no boolean, just plain bool)

Reply:
Have added other combinations